pkg/execute/executor.go

```diff
@@ -368,31 +345,12 @@ func (e *DefaultExecutor) findBotKubeVersion() (versions string) {
 	return fmt.Sprintf("K8s Server Version: %s\nBotKube version: %s", k8sVersion, botkubeVersion)
 }
 
-func (e *DefaultExecutor) runVersionCommand(args []string, clusterName string) string {
-	err := e.analyticsReporter.ReportCommand(e.platform, args[0])
+func (e *DefaultExecutor) runVersionCommand(cmd string) string {
+	err := e.analyticsReporter.ReportCommand(e.platform, cmd)
 	if err != nil {
-		// TODO: Return error when the DefaultExecutor is refactored as a part of https://github.com/kubeshop/botkube/issues/589
 		e.log.Errorf("while reporting version command: %s", err.Error())
 	}
 
-	checkFlag := false
-	for _, arg := range args {
-		if checkFlag {
-			if arg != clusterName {
-				return ""
-			}
-			checkFlag = false
-			continue
-		}
-		if strings.HasPrefix(arg, ClusterFlag.String()) {
-			if arg == ClusterFlag.String() {
-				checkFlag = true
-			} else if strings.SplitAfterN(arg, ClusterFlag.String()+"=", 2)[1] != clusterName {
-				return ""
-			}
-			continue
-		}
-	}
```

Comment:
Okay, so this is a part I asked a question about in previous comment:

> Should we ignore the execution if the cluster name doesn't match? (---cluster-name is provided)? I think we should 🤔

Reply:
it's already handled in line 142